### PR TITLE
Naming struct members the same way as class members.

### PR DESCRIPTION
**Pull request by remod**

### CODE REVIEW

**Review by pleemann — APPROVED** [review]

File: cppguide.html [hunk 1]
```diff
@@ -4725,14 +4725,15 @@ <h4 class="stylepoint_subsection">Class Data Members</h4>
...
-  string name;
-  <del>int num_entries;</del>
-  <strong>int numEntries;</strong>
-  static Pool&lt;UrlTableProperties&gt;* pool;
+are named like ordinary nonmember variables, <strong>but with a
+trailing underscore.</strong> <del>They do not have
+the trailing underscores that data members in classes have.</del></p>
+
+<pre>struct TableInfo {
+  <del>string table_name_;  // OK - underscore at end.</del>
```
Comment (hogabrie) [inline cppguide.html]: The idea was to clearly mark changes to the original google style guide. So do not edit without <del>del </del> and <strong>strong</strong> e.g. TableInfo change. and use the orignal variable names. Thanks for adapting though, should have told you.

Comment (remod) [reply cppguide.html]: Ok sorry, I copy/pasted the example for classes :)

Comment (remod) [reply cppguide.html]: Should be fine now. I am re-using the existing example, deleting/stronging the members.

Comment (hogabrie) [reply cppguide.html]: Thanks!

**Review by hogabrie — APPROVED** [review]

**Comment by remod** [comment]
Forgot to merge. :)

**Pull request merged**